packages/commons/src/interop-token/interopTokenService.ts

```diff
@@ -83,13 +84,15 @@ export class InteropTokenGenerator {
       kid: this.config.generatedKid,
     };
 
+    const duration = jwtDuration || this.config.generatedSecondsDuration;
```

Comment:
Using the "??" operator might be a safer option here

```suggestion
    const duration = jwtDuration ?? this.config.generatedSecondsDuration;
```